www/docs/en/9.x/platform_pinning/index.md

```diff
@@ -0,0 +1,52 @@
+---
+license: >
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+title: Platform Pinning
+toc_title: Platform pinning
+description: Cordova CLI pinned platform versions.
+---
+
+## Platform Pinning
+
+Cordova CLI pins the supported platform version in conjunction with the version of CLI that is being used.
```

Comment:
complicated. better way to say "each CLI version has a list of platform versions pinned in a file"?

Reply:
If written this way, I suspect users would want to see a link to this file?

The pinning is actually [here](https://github.com/apache/cordova-lib/blob/master/src/platforms/platformsConfig.json) in `cordova-lib`.

Any changes to make it less complicated is great, I only wanted to point out that if we start adding links to files in code, it means we would have to maintain this more frequently. 

On every release, we would need to update the link to the correct release version (not master) in LIB and anyone who does the future release of lib that includes pinning changes, they must know about this page and that it should as well be updated.

Reply:
I didn't suggest that you take this sentence, but asked for a better way to say what the sentence contains.

Reply:
Outdated: Refactored